lib/login_page.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:project_eureka_flutter/home_page.dart';
+import 'package:project_eureka_flutter/sign_in.dart';
+
+class LoginPage extends StatefulWidget {
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  //@overide
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        color: Colors.white,
+        child: Center(
+          child: Column(
+            mainAxisSize: MainAxisSize.max,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              //FlutterLogo(size: 150),
+              //SizedBox(height: 50),
+              _signInButton(context),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  void signInGoogle(result) {
+    signInWithGoogle().then((result) {
+      if (result != null) {
+        Navigator.push(
+          context,
+          MaterialPageRoute(
+            builder: (context) {
+              return HomePage();
+            },
+          ),
+        );
+      }
+    });
+  }
+
+  Widget _signInButton(BuildContext context) {
```

Comment:
Not as important but it's recommended to use `FlatButton` rather than `Widget` here to.